advanced/events.md

```diff
@@ -139,7 +139,7 @@ The following events are provided by the {{ site.productname }} editor.
 | PreProcess             | `{ node: Element, format: string }`                                                                                     | Fired before serializing a DOM node to HTML content.                                                                                                                       |
 | PostProcess            | `{ node: Element, format: string }`                                                                                     | Fired after serializing a DOM node to HTML content.                                                                                                                        |
 | SetAttrib              | `{ attrElm: Element, attrName: string, attrValue: string }`                                                             | Fired when an attribute is updated using the editor [DOMUtils API]({{ site.baseurl }}/api/tinymce.dom/tinymce.dom.domutils/).                                                                                                          |
-| ResizeEditor           | N/A                                                                                                                     | Fired when the editor is resized, either by the resize handles or the auto-resize plugin.                                                                                  |
+| ResizeEditor           | N/A                                                                                                                     | Fired when the editor is resized, either by the resize handles or the auto-resize plugin. As of {{site.productname}} 5.10.6, this event is also fired fullscreen mode is toggled. Toggling fullscreen mode requires the `fullscreen` plugin to be enabled.                                                                                 |
```

Comment:
This seems to be missing a word? I think it should be this:
```suggestion
| ResizeEditor           | N/A                                                                                                                     | Fired when the editor is resized, either by the resize handles or the auto-resize plugin. As of {{site.productname}} 5.10.6, this event is also fired when fullscreen mode is toggled. Toggling fullscreen mode requires the `fullscreen` plugin to be enabled.                                                                                 |
```
I'm also not convinced the last sentence is needed 🤷